frontend/talentsearch/src/js/components/request/CreateRequest.tsx

```diff
@@ -369,16 +368,14 @@ export const RequestForm: React.FunctionComponent<RequestFormProps> = ({
               data-h2-text-align="base(center) p-tablet(left)"
               data-h2-flex-item="base(1of1) p-tablet(1of2)"
             >
-              <Button
+              <Link
                 color="primary"
                 mode="outline"
                 data-h2-margin="base(0, x.5, 0, 0)"
-                onClick={() => {
-                  // Save the initial search form values to the state so they are available to user when click back.
-                  pushToStateThenNavigate<BrowserHistoryState>(paths.search(), {
-                    ...state,
-                    initialValues: searchFormInitialValues,
-                  });
+                href={paths.search()}
+                state={{
```

Comment:
This could probably be spun off into its own issue but changing languages loses this state.

https://user-images.githubusercontent.com/8978655/202283547-44edc346-4f96-4eaa-9d0d-2b385b878f18.mp4

Reply:
Yeah, I'm going to assume that is because we are storing it in the route state and a refresh would likely clear that. Maybe it might be best to store these in search params instead?g